scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ServerNameExtension.java

```diff
@@ -1,15 +1,15 @@
 /*******************************************************************************
  * Copyright (c) 2016 Bosch Software Innovations GmbH and others.
- * 
+ *
```

Comment:
Can you please leave all comments intact? I assume you are using some code formatting rules that eliminate these whitespaces. I have checked the Californium code formatting rules but couldn't find such a setting that would be responsible for it. Are you using Eclipse at all?